### DISCUSSION TITLE
Carbon input addons

### DISCUSSION THREAD

**Discussion by skateman** [data-driven-forms/react-forms#912] [Ideas]
I talked to some carbon UX people about :point_up: and they are okay with it, however, they aren't implemented as a drop-in solution in their react components package :confused: 

![image](https://user-images.githubusercontent.com/649130/101772987-708e5100-3aec-11eb-9988-e3b5058c56c3.png)

@Hyperkid123 @rvsia would it be possible to lego it together here using existing carbon components?

**Comment by rvsia** [comment]
Hello @skateman,

I think you should go directly to Carbon team and let them implement this feature in their library as we don't have full control over all components. For example, our `TextField` is just Carbon `TextInput` - there is no simple way how to add buttons, etc. next to the input.

And because of non-modularity of Carbon components, it's not possible to construct custom `TextField` from basic components, because these components do not exist.

**Comment by skateman** [comment]
I think we can close this, I found another way...
